backend/src/main/java/com/github/sirtobiwan/backend/security/DtoVaccineUser.java

```diff
@@ -0,0 +1,9 @@
+package com.github.sirtobiwan.backend.security;
+
+import org.springframework.data.annotation.Id;
+
+public record DtoVaccineUser(@Id
```

Comment:
Why do you need the `@Id` annotation here?
Also your DTO should have less attributes than the real User class. Otherwise it's just a copy of the VaccineUser class

Reply:
true, the id is not necessary. Thanks